src/win/fs.c

```diff
@@ -113,9 +113,44 @@ const WCHAR LONG_PATH_PREFIX_LEN = 4;
 const WCHAR UNC_PATH_PREFIX[] = L"\\\\?\\UNC\\";
 const WCHAR UNC_PATH_PREFIX_LEN = 8;
 
+static DWORD uv__file_symlink_flags;
+
+static int uv__usermode_symlink_available(void) {
```

Comment:
wow, this is really involved :-(

What happens if we put the flag value allways? Can we identify the failure and enable a fallback? That's mostly how we do feature detection on the Unix side.